cypress/e2e/DiagramsPage/DeleteDiagrams.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Test diagrams page: delete diagram
```

Comment:
put this in commit like "Update e2e test"

Reply:
done